rocky/katalogus/templates/partials/plugin_tile.html

```diff
@@ -24,6 +26,21 @@
             </div>
         {% endif %}
         <span class="plugin-author-link de-emphasized"><a target="_blank" href="https://openkat.nl" rel="noopener noreferrer">OpenKAT</a></span>
+        {% if show_report_types %}
+            <p>
+                {% for plugin_id, report_types in plugin_report_types.items %}
+                    {% if plugin_id == plugin.id %}
+                        {% if report_types|length > 1 %}
+                            {% translate "Required for" %} {{ report_types|length }} {% translate "report types" %}
```

Comment:
I think it would be nicer if this was pluralized. You can use the `blocktranslate` to achieve this.

Reply:
No longer applicable, code has been changed/removed.